src/components/common/MessageListItem.js

```diff
@@ -0,0 +1,25 @@
+import React, {PropTypes} from 'react';
+import { isMyMessage } from '../../utils/uiHelper/ChatHelper';
+
+const MessageListItem = ({ message }) => {
+
```

Comment:
You can refactor this component to something like this:
```
const MessageListItem = ({ message }) => (
  <div className={isMyMessage(message) ? "bubbleme" : "bubbleyou"}>
    {message.text}
  </div>
);
```

Really simple, right?